usagov/usagov-2021
# USAGOV-1837: Cypress-Footer-Updates: Footer Regression Check Updates

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1837

## Description

Updates the footer test for regression testing. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- Follow the steps in the ReadMe (https://github.com/usagov/usagov-2021?tab=readme-ov-file#automated-tests-setup-cypress) to set up Cypress for either headless or interactive tests. 
- Run footer.cy.js (automated_tests/e2e-cypress/cypress/e2e/regression_testing/footer.cy.js)
- Make sure tests pass

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
